spec/setup_acceptance_node.pp

```diff
@@ -5,7 +5,7 @@
 }
 
 class { 'candlepin::repo':
-  version => pick(fact('candlepin_version'), 'nightly'),
+  version => pick(fact('candlepin_version'), '4.3'),
```

Comment:
Why 4.3 instead of nightly?

Reply:
The nightly one is empty right now, as we have no dedicated candlepin test and publish pipeline yet

Reply:
Basically yes, I wanted to test and for this to work from a known working point in time. Then to use this new feature to enable the testing pipelines.

Reply:
I have a working nightly repository for EL8 now, so I reverted this change back.